x-pack/test/functional/services/observability/alerts/common.ts

```diff
@@ -270,6 +271,15 @@ export function ObservabilityAlertsCommonProvider({
     return actionsOverflowButtons[index] || null;
   };
 
+  const getAlertStatValue = async (testSubj: string) => {
```

Comment:
@claudiopro I suggest we keep using the correct terminology. Could you rename to getRuleStatValue